### PR TITLE
Modular rework of ipfs.io for w3dt positioning and PMF alignment

### PR DESCRIPTION
**Pull request by jessicaschilling**
## Modular rework of ipfs.io for w3dt positioning and PMF alignment

This project will enable and empower both prospective developers and the internal PL team by the following means:
- Providing an accurate, up-to-date, clearly-pathed entry point for prospective w3dt developers that ...
     - illustrates where IPFS fits into the w3dt stack as a whole
     - offers clear value propositions of IPFS (as part of w3dt) for our most important use cases
     - provides a concrete on-ramp for getting started building
- Creating a baseline entry point to our PMF funnel that can be iteratively measured, improved, and re-measured

### CODE REVIEW

**Comment by jessicaschilling** [comment]
cc @atopal

File: proposals/ipfsio-modular-rework.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+## Purpose &amp; impact 
+#### Background &amp; intent
+_Describe the desired state of the world after this project? Why does that matter?_
+
+This project will enable us to finally have a concise, up-to-date, clearly-pathed entry point for prospective w3dt developers that illustrates where IPFS fits into the w3dt stack as a whole, offers clear value propositions of IPFS (as part of w3dt) for our most important use cases, and provides a concrete on-ramp for getting started building.
+
+This matters because:
+- It enables us to have an entry point to our PMF measurement funnel that doesn't itself detract or muddy our metrics
+- With the Jan 2020 paradigm shift, we now have *massive* "comms debt" about how IFPS fits into the w3dt stack as a whole
```
Comment (alanshaw) [inline proposals/ipfsio-modular-rework.md:31]: Classic typo ;)

```suggestion
- With the Jan 2020 paradigm shift, we now have *massive* "comms debt" about how IPFS fits into the w3dt stack as a whole
```

Comment (atopal) [reply proposals/ipfsio-modular-rework.md:31]: actually 2021 ;)

File: proposals/ipfsio-modular-rework.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+## Purpose &amp; impact 
+#### Background &amp; intent
+_Describe the desired state of the world after this project? Why does that matter?_
+
+This project will enable us to finally have a concise, up-to-date, clearly-pathed entry point for prospective w3dt developers that illustrates where IPFS fits into the w3dt stack as a whole, offers clear value propositions of IPFS (as part of w3dt) for our most important use cases, and provides a concrete on-ramp for getting started building.
+
+This matters because:
+- It enables us to have an entry point to our PMF measurement funnel that doesn't itself detract or muddy our metrics
+- With the Jan 2020 paradigm shift, we now have *massive* "comms debt" about how IPFS fits into the w3dt stack as a whole
```
Comment (jessicaschilling) [inline proposals/ipfsio-modular-rework.md:31]:
```suggestion
- With the Jan 2021 paradigm shift, we now have *massive* "comms debt" about how IPFS fits into the w3dt stack as a whole
```

**Review by jessicaschilling — COMMENTED** [review]
Typos fixed.

**Comment by jessicaschilling** [comment]
@atopal @momack2 As discussed with @atopal, will break this PR into separate individual parts suitable for end-to-end work in 2-4 weeks each. Will reference here and close this PR once finished. Thanks.

**Comment by jessicaschilling** [comment]
Closing this issue in favor of separate PRs for work currently underway:

- #72 Interactive ecosystem directory
- #73 "Builders Series" videos
- #74 IPFS blog enhancements
- #75 "Lift-and-shift" replatforming of ipfs.io

This will enable more granular tracking of these smaller projects in the short term. Thank you.

cc @atopal @momack2

**Pull request closed without merging**